bpf/bpf_sock.c

```diff
@@ -160,7 +160,11 @@ sock4_skip_xlate(struct lb4_service *svc, __be32 address)
 {
 	if (lb4_to_lb6_service(svc))
 		return true;
-	if (lb4_svc_is_external_ip(svc) ||
+
+	if (
+#ifndef DISABLE_EXTERNAL_IP_MITIGATION
+		lb4_svc_is_external_ip(svc) ||
+#endif
```

Comment:
As the BPF stylechecker is complaining about this pattern, one suggestion to make it a bit cleaner:

```suggestion
	if ((lb4_svc_is_external_ip(svc) && !is_defined(DISABLE_EXTERNAL_IP_MITIGATION))) ||
```

Reply:
Fixed